<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>375</width>
    <height>667</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>375</width>
    <height>667</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>375</width>
    <height>667</height>
   </size>
  </property>
  <property name="font">
   <font>
    <pointsize>13</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow#MainWindow{
background:rgb(51, 51, 51)
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QDoubleSpinBox" name="WeightInput">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>180</y>
      <width>71</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>120</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel#label_2{
color:rgb(230, 230, 230)
}</string>
    </property>
    <property name="text">
     <string>Height(ft)</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>190</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel#label_3{
color:rgb(230, 230, 230)
}</string>
    </property>
    <property name="text">
     <string>Weight(kg)</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="AgeInput">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>250</y>
      <width>71</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>260</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel#label_4{
color:rgb(230, 230, 230)
}</string>
    </property>
    <property name="text">
     <string>Age</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>90</y>
      <width>201</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel#label_5{
color:rgb(230, 230, 230)
}</string>
    </property>
    <property name="text">
     <string>Email</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>160</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel#label_6{
color:rgb(230, 230, 230)
}</string>
    </property>
    <property name="text">
     <string>Phone Number</string>
    </property>
   </widget>
   <widget class="QPushButton" name="CreateAccount">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>410</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Create Account</string>
    </property>
   </widget>
   <widget class="QPushButton" name="AlreadyHaveAnAccount">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>480</y>
      <width>191</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Already have an Account?</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>240</y>
      <width>211</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel#label_7{
color:rgb(230, 230, 230)
}</string>
    </property>
    <property name="text">
     <string>Username</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>320</y>
      <width>211</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel#label_8{
color:rgb(230, 230, 230)
}</string>
    </property>
    <property name="text">
     <string>Password</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>0</y>
      <width>311</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>53</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel#label{
color:rgb(204, 255, 102)
}
</string>
    </property>
    <property name="text">
     <string>ùîæùï™ùïû ùïéùïöùï´ùïíùï£ùïï</string>
    </property>
   </widget>
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>350</y>
      <width>121</width>
      <height>161</height>
     </rect>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>-10</x>
      <y>80</y>
      <width>118</width>
      <height>3</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">Line#line{
background:rgb(76, 76, 76)
}</string>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>80</y>
      <width>118</width>
      <height>3</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">Line#line_2{
background:rgb(76, 76, 76)
}</string>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_3">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>80</y>
      <width>118</width>
      <height>3</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">Line#line_3{
background:rgb(76, 76, 76)
}</string>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_4">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>80</y>
      <width>118</width>
      <height>3</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">Line#line_4{
background:rgb(76, 76, 76)
}</string>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QComboBox" name="HeightInput">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>80</width>
      <height>26</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>5</width>
      <height>0</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777194</width>
      <height>16777186</height>
     </size>
    </property>
    <property name="currentText">
     <string>4'10</string>
    </property>
    <property name="currentIndex">
     <number>10</number>
    </property>
    <property name="maxVisibleItems">
     <number>5</number>
    </property>
    <property name="maxCount">
     <number>999999984</number>
    </property>
    <property name="insertPolicy">
     <enum>QComboBox::InsertAtTop</enum>
    </property>
    <property name="sizeAdjustPolicy">
     <enum>QComboBox::AdjustToContentsOnFirstShow</enum>
    </property>
    <property name="duplicatesEnabled">
     <bool>false</bool>
    </property>
    <item>
     <property name="text">
      <string>4'0</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>4'1</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>4'2</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>4'3</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>4'4</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>4'5</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>4'6</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>4'7</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>4'8</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>4'9</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>4'10</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>4'11</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>5'0</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>5'1</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>5'2</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>5'3</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>5'4</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>5'6</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>5'7</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>5'8</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>5'9</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>5'10</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>5'11</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>6'0</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>6'1</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>6'2</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>6'3</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>6'4</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>6'5</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>6'6</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>6'7</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>6'8</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>6'9</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>6'10</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>6'11</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>7'0</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>7'1</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>7'2</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>7'3</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>7'4</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>7'5</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>7'6</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>7'7</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>7'8</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>7'9</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>7'10</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>7'11</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>8'0</string>
     </property>
    </item>
   </widget>
   <widget class="QLineEdit" name="PasswordInput">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>350</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="UsernameInput">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>270</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="PhoneNumberInput">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>190</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="EmailInput">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>120</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>375</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
